.github/workflows/release-flux-artifact.yml

```
@@ -0,0 +1,95 @@
+              --path="./flux/${artifact}" \
+              --source="$(git config --get remote.origin.url)" \
+              --revision="$(git branch --show-current)/$(git rev-parse HEAD)" || { echo "Failed to push $artifact" >&2; exit 1; }
+              --creds="${creds}"
+            flux tag artifact oci://${REGISTRY}/manifests/infra/${artifact}:$(git rev-parse --short HEAD) \
+              --provider=azure \
+              --tag latest \
+              --creds="${creds}" || { echo "Failed to tag $artifact with latest" >&2; exit 1; }
+          done
+  tag-release:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && github.event.action == 'created' && startsWith(github.event.release.tag_name, github.env.RELEASE_TAG_PREFIX)
```

actionlint (1.7.7)

receiver of object dereference "release_tag_prefix" must be type of object but got "string"

(expression)